modules/message.sh

```diff
@@ -64,6 +64,40 @@ function message {
           ;;
       esac
       ;;
+    hangouts)
+      case "$2" in
+        login)
+          #$(pip3 install hangups)
+          #$(hangups --manual-login)
+          if ! [ -e /root/cli/templates/hangups ]; then
```

Comment:
this is not the correct directory if using the npm package, this will work only if you clone the cli repo in the home directory
consider using `"$(which treehouses | sed 's/bin\/treehouses//')lib/node_modules/@treehouses/cli/templates"`